docs/diagnostics/PX1007.md

```diff
@@ -28,14 +36,13 @@ The diagnostic is currently displayed for the following public entities:
  - DAC property fields
 
 The diagnostic is not displayed for the following items: 
- - Entities with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
+ - Entities with the `PXInternalUseOnly` or `Obsolete` attribute.
+   ** Note: ** The `PXHidden` attribute does not suppress the diagnostic.
  - Members of public entities (except nested public types and DAC property fields).
  - All members of a public entity if the entity has the `exclude` XML tag or is marked with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
```

Comment:
The part with attributes is not precise and became outdated. It should be updated with something like this:
```csharp
 - All members of a public entity if the entity has the `exclude` XML tag or is marked with the `PXInternalUseOnly`, or `Obsolete` attribute. You can also exclude DACs from the documentation by marking them with `PXHidden` attribute.
```